dough-reflection/src/main/java/io/github/bakedlibs/dough/reflection/ReflectionUtils.java

```diff
@@ -267,10 +264,8 @@ public static Field getField(@Nonnull Class<?> c, @Nonnull String field) throws
      * @throws UnknownServerVersionException
      *             If the {@link MinecraftVersion} was unable to be determined
      */
-    public static Class<?> getNetMinecraftClass(@Nonnull String name) throws ClassNotFoundException, UnknownServerVersionException {
-        MinecraftVersion version = MinecraftVersion.of(Bukkit.getServer());
-        String suffix = version.isAtLeast(1, 17) ? "" : CURRENT_VERSION + '.';
-        return Class.forName("net.minecraft." + suffix + name);
+    public static @Nonnull Class<?> getNetMinecraftClass(@Nonnull String name) throws ClassNotFoundException, UnknownServerVersionException {
+        return Class.forName("net.minecraft." + getVersionSpecificPackage(true) + name);
```

Comment:
I wonder if it would make sense to cache the string package string that way we won't need to parse the version-specific package every method call.

Reply:
Not sure what you mean... the entire point of this method is to cache this package string.

Reply:
I'm an idiot, don't mind me :P